Source/WebKit/WPEPlatform/wpe/WPESettings.cpp

```diff
@@ -92,14 +92,18 @@ struct _WPESettingsPrivate {
             { WPE_SETTING_FONT_NAME, G_VARIANT_TYPE_STRING, g_variant_ref_sink(g_variant_new_string("Sans 10")) },
             { WPE_SETTING_DARK_MODE, G_VARIANT_TYPE_BOOLEAN, g_variant_ref_sink(g_variant_new_boolean(false)) },
             { WPE_SETTING_DISABLE_ANIMATIONS, G_VARIANT_TYPE_BOOLEAN, g_variant_ref_sink(g_variant_new_boolean(false)) },
+            { WPE_SETTING_DRM_SCALE, G_VARIANT_TYPE_DOUBLE, g_variant_ref_sink(g_variant_new_double(1.0)) },
```

Comment:
This should not be registered here, but it's already registered in WPEDisplayDRM.cpp The definition should be moved to DRM too, but that can be done in a different patch.

Reply:
PR updated according to this comment.